models/Tool/SettingsStore.php

```diff
@@ -21,7 +21,7 @@
 use Pimcore\Model\Tool\SettingsStore\Dao;
 
 /**
- * @method Dao getById(string $id)
+ * @method Dao getById(string $id, ?string $scope)
```

Comment:
`Dao::getById()` actually sets the data on the model and then returns `bool`.
```suggestion
 * @method bool getById(string $id, ?string $scope)
```